.github/workflows/lint.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  super-linter:
+    name: Run Super-Linter
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Run Super-Linter
+        uses: github/super-linter@v3.17.0
+        env:
+          VALIDATE_ALL_CODEBASE: false
```

Comment:
What pieces of code failed without this option? I'd like the linter to check all code to avoid situation when editing only a whitespace in a file causes a linter failure. It discourages the developer from refactoring

Reply:
I just added it by default, I will check if setting this option to true will cause fail.

Reply:
Done